tests/models/auto/test_modeling_tf_auto.py

```diff
@@ -281,7 +281,7 @@ def test_revision_not_found(self):
     def test_model_file_not_found(self):
         with self.assertRaisesRegex(
             EnvironmentError,
-            "hf-internal-testing/config-no-model does not appear to have a file named tf_model.h5",
+            "hf-internal-testing/config-no-model does not appear to have a file named pytorch_model.bin",
```

Comment:
Good to see this fixed the CI, however this is somehow confusing: it is `TFAutoModel`, but seeing the expected error being `pytorch_model.bin` here blows my mind.

(I know the full error is `hf-internal-testing/config-no-model does not appear to have a file named pytorch_model.bin, tf_model.h5 or model.ckpt`)